output/elasticsearch/output.go

```diff
@@ -108,10 +108,17 @@ func (es *ElasticSearchOutput) Start() error {
 					es.logger.Errorf("failed to serialize event: %v", err)
 					continue
 				}
+				var rawIndex string
+				// If the event metadata contains a raw_index field, attach
+				// that to the indexer item to override the global default.
+				if indexField := event.GetMetadata().GetData()["raw_index"]; indexField != nil {
```

Comment:
I'm assuming if the event doesn't have `raw_index` this will fall back to the one when we did the NewBulkIndexer.  I'm wondering if we should get rid of the default index and fail on the event if it doesn't have the `raw_index`.  Or do you want to keep the default for a little bit longer for dev purposes?

Reply:
I think this should be an error eventually but yea I'm reluctant to remove the fallback until we've observed things getting consistently ingested to the right place on live traffic :-)